tekton/resources/release/github_release.yaml

```diff
@@ -50,11 +50,10 @@ spec:
       script: |
         #!/bin/bash
         set -ex
-        p=$(basename $PROJECT)
-        TEKTON_PROJECT=${p^}
+        TEKTON_PROJECT=$(basename $PROJECT)
```

Comment:
FYI: If you wanted to do fancy you could have maybe done also : 

```${PROJECT##*/}```

but `basename` works too 😅